File: snippets/python_snippets/receive_payment.py
import breez_sdk
import logging

sdk_services = breez_sdk.BlockingBreezServices

def receive_payment():
    try:
        # ANCHOR: receive-payment
        req = breez_sdk.ReceivePaymentRequest(
        amount_msat=300000,
        description="Invoice for 300 000 sats")
        result = sdk_services.receive_payment(req)
        # ANCHOR_END: receive-payment
        return result
    except Exception as error:
        logging.error(error)
        raise

File: snippets/kotlin_mpp_lib/shared/src/commonMain/kotlin/com/example/kotlinmpplib/ReceivePayment.kt
package com.example.kotlinmpplib

import breez_sdk.*
class ReceivePayment {
    fun receivePayment(sdk: BlockingBreezServices) {
        // ANCHOR: receive-payment
        try {
            val receivePaymentResponse = sdk.receivePayment(ReceivePaymentRequest(
                3_000_000.toULong(),
                "Invoice for 3000 sats",
            ))

            val invoice = receivePaymentResponse.lnInvoice;
        } catch (e: Exception) {
            // handle error
        }
        // ANCHOR_END: receive-payment
    }
}